wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/ProfileViewController.swift

```diff
@@ -47,26 +47,17 @@ extension ZMConversationType {
 }
 
 final class ProfileViewController: UIViewController {
-    let viewModel: ProfileViewControllerViewModel
     weak var viewControllerDismisser: ViewControllerDismisser?
+    weak var delegate: ProfileViewControllerDelegate?
 
+    private var viewModel: ProfileViewControllerViewModeling
```

Comment:
```suggestion
    private let viewModel: ProfileViewControllerViewModeling
```

Do we replace the viewModel at some point?

Reply:
No but the compiler didn't like that I was setting the delegate property directly. I chose to use a `setDelegate` method instead

Reply:
https://github.com/wireapp/wire-ios/pull/1365/commits/c8e57081d54b03ce21f7cd4b9c318faab9519978